migration/migrator/migrations/system/20200328225850_docker_ui.py

```diff
@@ -0,0 +1,29 @@
+"""Migration for the Submitty system."""
+import os
+import pwd
+import grp
+
+def up(config):
+    """
+    Run up migration.
+
+    :param config: Object holding configuration details about Submitty
+    :type config: migrator.config.Config
+    """
+   
+    #add submitty_cgi to docker group
+    os.system("usermod -a -G docker " + config.submitty_users['cgi_user'] )
+    #allow php to read the autograding container
+
+    tgt_dir = os.path.join(config.submitty['submitty_install_dir'], 'config', 'autograding_containers.json')
+    os.chown(tgt_dir, config.submitty_users['daemon_uid'], config.submitty_users['php_gid'])
```

Comment:
What is the permissions for this file before this and where is it set? In CONFIGURE_SUBMITTY.py?

Reply:
I can't find where it gets created but there's a migration for making it and the files `CONFIGUIRE_SUBMITTY.py` and `update_and_install_workers.py` use it, its owned by the daemon it gets written to by having instructors modify the file manually